Findingcircles.py

```diff
@@ -0,0 +1,244 @@
+    innercircle = np.empty(3, dtype=int)
+    outercircle = get_circle(intimage, 200, params_big)
+    innercircle = get_circle(intimage, 300, params_small)
+
+    if do_plot == 1:
+        fig, (ax1, ax2) = plt.subplots(1, 2)
+        for (x, y, r) in outercircle:
+            # draw the circle in the output image, then draw a rectangle
+            # corresponding to the center of the circle
+            cv2.circle(intimage, (x, y), r, (128, 128, 128), 4)
+            cv2.rectangle(intimage, (x - 5, y - 5), (x + 5, y + 5), (128, 128, 128), -1)
+        ax1.imshow(intimage, origin='lower')
+        for (x, y, r) in innercircle:
+            # draw the circle in the output image, then draw a rectangle
+            # corresponding to the center of the circle
+            cv2.circle(intimage, (x, y), r, (128, 128, 128), 4)
+            cv2.rectangle(intimage, (x - 5, y - 5), (x + 5, y + 5), (128, 128, 0), -1)
+        ax2.imshow(intimage, origin='lower')
+        fig.savefig(path+'/detail5.png')
+    # print("outer circle is", outercircle, "inner circle is", innercircle, flush=True)
+    # print(params_big, params_small, flush=True)
+    return outercircle, innercircle
+
+
+def _getEfdData(client, dataSeries, startTime, endTime):
+    import asyncio
+    """A synchronous warpper for geting the data from the EFD.
+    This exists so that the top level functions don't all have to be async def.
+    curtesy of Merlin Levine-Fisher.
+    """
+    loop = asyncio.get_event_loop()
+    return loop.run_until_complete(client.select_time_series(dataSeries, ['*'], startTime, endTime))
+
+
+def getEFDinfo(dayObs, seqNum):
+    """ Wrapper that grabs the EFD info for each sequence, currently this
+    wrapper does not work, there seems to be an issue with asyncio."""
+    from astropy.time import Time, TimeDelta
+    from lsst_efd_client import EfdClient
+    # import pandas as pd
+
+    client = EfdClient('summit_efd')
+    butler = dafPersist.Butler('/project/shared/auxTel/rerun/quickLook')
+
+    dataId = {'dayObs': dayObs, 'seqNum': seqNum}
+    expId = butler.queryMetadata('raw', 'expId', **dataId)[0]
+
+    tStart = butler.queryMetadata('raw', ['DATE'], detector=0, expId=expId)  # Get the data
+    t_start = Time(tStart, scale='tai')[0]
+    t_end = t_start + TimeDelta(1, format='sec')
+
+    # Get the reported position
+    # hex_position = client.select_time_series("lsst.sal.ATHexapod.positionStatus", ['*'],t_start, t_end)
+    # hex_position = _getEfdData(client, "lsst.sal.ATHexapod.positionStatus", t_start, t_end)
+    # Wild attempt from my side:
+    hex_position = client.select_time_series("lsst.sal.ATHexapod.positionStatus", ['*'], t_start, t_end)
+    print(hex_position.head())
+    # This dictionary gives the hexapod names and indices
+    # units for x,y,z in mm and u,v,w in degrees, according to
+    # https://ts-xml.lsst.io/sal_interfaces/ATHexapod.html#positionupdate.
+
+    names = {'u': 3, 'v': 4, 'w': 5, 'x': 0, 'y': 1, 'z': 2}
```

Comment:
You don't seem to have done anything regarding the old comment here. You're still not using the values anywhere, and this could just be a list of names, right?